e2e-tests/cypress/support/insurance/complete-company-details.js

```diff
@@ -0,0 +1,10 @@
+import { companyDetails } from '../../e2e/pages/your-business';
+import { submitButton, yesRadioInput } from '../../e2e/pages/shared';
+import { COMPANIES_HOUSE_NUMBER } from '../../../constants';
+
+export default () => {
+  companyDetails.companiesHouseSearch().clear().type(COMPANIES_HOUSE_NUMBER);
+  yesRadioInput().first().click();
+  yesRadioInput().eq(1).click();
```

Comment:
Are we going to have more descriptive radios? Think we mentioned in a previous PR, concious of more instances of this being introduced. Could have e.g `xFieldYesRadioInput`  Not a blocker

Reply:
how would we do this as we have a lot of files using `yesRadioInput`

Reply:
As part of `companyDetailsPage`